k8s/tools/cleanup-pubsub-gke.sh

```diff
@@ -101,8 +101,12 @@ if [[ "$*" != *--no-filestore* ]] ; then
   gcloud -q --project $DEVSHELL_PROJECT_ID filestore instances delete $FILESTORE_NAME --zone $ZONE
 fi
 if [[ "$*" != *--no-dfdewey* ]] ; then
```

Comment:
Small nit, but a comment right below here noting something like "Cleaning up DfDewey deployment" can help differentiate the Dfdewey filestore cleanup vs Turbinia filestore cleanup